src/sdk/model/company/__json__/o365-org-create-request-json.ts

```diff
@@ -0,0 +1,8 @@
+/* istanbul ignore next */
```

Comment:
No need for the istanbul comment here, only applies to methods and classes. Need to just add normal docs for this interface

Reply:
fixed!